tools/build-tools/src/fluidBuild/tasks/leaf/tscTask.ts

```diff
@@ -29,6 +29,14 @@ export class TscTask extends LeafTask {
     private _projectReference: TscTask | undefined;
     private _sourceStats: fs.Stats[] | undefined;
 
+    public matchTask(command: string, options?: any): LeafTask | undefined {
```

Comment:
adding type here would be nice, rather than options any

Reply:
done.